src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  max-width: 100%;
+  padding: 0 50px;
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+    object-fit: cover;
+  }
+}
+
+.nav-block {
+  display: flex;
+  align-items: center;
+  color: #000;
+
+  &__list {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 20px;
+    margin: 0;
+    padding: 0;
```

Comment:
Please reset default margins and padding in `reset.scss` file for all elements at once